### ISSUE TITLE
Fastqc 0.11.2 - naming issue

### ISSUE THREAD

**Issue by scholtalbers** [bioconda/bioconda-recipes#7060]
Recently I added Fastqc version 0.11.2: https://github.com/bioconda/bioconda-recipes/tree/master/recipes/fastqc/0.11.2

This apparently resulted in the following packages: https://anaconda.org/bioconda/fastqc/files?version=0.11.2

When one now however wants to install fastqc=0.11.2 on a linux machine one gets the older package, which does not have java dependency.. :
```
$ conda create -n fastqc-0.11.2 fastqc=0.11.2
Fetching package metadata ...............
Solving package specifications: .

Package plan for installation in environment ~/conda-envs/fastqc-0.11.2:

The following NEW packages will be INSTALLED:

    fastqc: 0.11.2-0 bioconda

Proceed ([y]/n)? 
```

Can this be resolved?

**Comment by mbargull** [comment]
Oh, I didn't notice `linux-64/fastqc-0.11.2-0.tar.bz2` was available when reviewing #6992... We should have used
```yaml
build:
  number: 1
```
there. But since the prior build lacks a required dependency anyway, we can flag it as broken nonetheless. @johanneskoester: can you add the `broken` label to https://conda.anaconda.org/bioconda/linux-64/fastqc-0.11.2-0.tar.bz2?

**Comment by daler** [comment]
@mbargull done. Thanks for pointing this out, sorry for the delay.
